docs/src/test/java/jdocs/testkit/TestKitDocTest.java

```diff
@@ -31,15 +32,6 @@
 
 import com.typesafe.config.ConfigFactory;
 
-import org.apache.pekko.actor.AbstractActorWithTimers;
-import org.apache.pekko.actor.ActorKilledException;
-import org.apache.pekko.actor.ActorRef;
-import org.apache.pekko.actor.ActorSystem;
-import org.apache.pekko.actor.Kill;
-import org.apache.pekko.actor.PoisonPill;
-import org.apache.pekko.actor.Props;
-import org.apache.pekko.actor.Terminated;
```

Comment:
Could we keep these imports? This class is part of our docs and it useful to be explicit in the imports.